src/components/Preferences/LocalIdentities.js

```diff
@@ -55,33 +55,48 @@ const LocalIdentities = React.memo(
         /* nothing was updated */
       }
     })
-    const allSelected = Array.from(addressesSelected.values()).every(v => v)
-    const someSelected = Array.from(addressesSelected.values()).some(v => v)
+    let allSelected = true
+    Object.keys(localIdentities).forEach(key => {
+      allSelected = allSelected && addressesSelected.get(localIdentities[key])
+    })
+    let someSelected = true
+    Object.keys(localIdentities).forEach(key => {
+      someSelected = someSelected || addressesSelected.get(localIdentities[key])
+    })
```

Comment:
Small nit, I think both of these could still be done with `every` and `none`:

```js
const allSelected = Object.keys(localIdentities).every(key => addressesSelected.get(localIdentities[key]))
const someSelected = Object.keys(localIdentities).some(key => addressesSelected.get(localIdentities[key]))
```

Reply:
Thank you, yes!